src/Component/Page/Home.purs

```diff
@@ -0,0 +1,208 @@
+      case tab of 
+        FeedTab -> pure a -- TODO
+        GlobalTab -> eval $ LoadArticles noArticleParams a
+        TagTab tag -> eval $ LoadArticles noArticleParams{ tag = Just tag } a
+  
+
+  render :: State -> H.ParentHTML Query (Const Void) Void m
+  render state@{ tags, articles} =
+    HH.div_
+    [ header Home Navigate
+    , HH.div
+      [ HP.class_ CC.homePage ]
+      [ banner
+      , HH.div
+        [ HP.classes [ HB.container, CC.page ] ]
+        [ HH.div
+          [ HP.class_ HB.row ]
+          [ mainView state
+          , HH.div
+            [ HP.class_ HB.colMd3 ]
+            [ HH.div
+              [ HP.class_ CC.sidebar ]
+              [ HH.p_ [ HH.text "Popular Tags" ]
+              , renderTags tags
+              ]
+            ]
+          ]
+        ]
+      ]
+    , footer
+    ]
+
+  banner :: forall p i. HH.HTML p i
+  banner =
+    HH.div
+    [ HP.class_ CC.banner ]
+    [ HH.div
+      [ HP.class_ HB.container ]
+      [ HH.h1
+        [ HP.class_ CC.logoFont ]
+        [ HH.text "conduit" ]
+      , HH.p_ [ HH.text "A place to share your knowledge." ]
+      ]
+    ]
+
+  mainView :: forall p. State -> H.HTML p Query
+  mainView state =
+    HH.div
+    [ HP.class_ HB.colMd9 ]
+    [ HH.div
+      [ HP.class_ CC.feedToggle ]
+      [ HH.ul
+        [ HP.classes [ HB.nav, HB.navPills, CC.outlineActive ] ]
+        [ yourFeedTab state.tab
+        , globalFeedTab state.tab
+        , tagFilterTab state.tab
+        ]
+      ]
+    , articleList state.articles
+    ]
+
+  yourFeedTab :: forall p i. Tab -> HH.HTML p i
+  yourFeedTab tab =
+    let modifiers = case tab of
+          FeedTab -> [ HB. active ]
+          _ -> []
+    in
+     HH.li
+     [ HP.class_ HB.navItem ]
+     [ HH.a
+       [ HP.classes $ [ HB.navLink ] <> modifiers ]
+       [ HH.text "Your Feed" ]
+     ]
+  
+  globalFeedTab :: forall p. Tab -> H.HTML p Query
+  globalFeedTab tab =
+    let modifiers = case tab of
+          GlobalTab -> [ HB. active ]
+          _ -> []
+    in
+     HH.li
+     [ HP.class_ HB.navItem ]
+     [ HH.a
+       [ HP.classes $ [ HB.navLink ] <> modifiers
+       , HE.handler click $ HE.input $ ShowTab GlobalTab
+       ]
+       [ HH.text "Global Feed" ]
+     ]
+
+  tagFilterTab :: forall p i. Tab -> HH.HTML p i
+  tagFilterTab = case _ of
+    TagTab tag ->
+      HH.li
+      [ HP.class_ HB.navItem ]
+      [ HH.a
+        [ HP.classes $ [ HB.navLink, HB.active ] ]
+        [ HH.i [ HP.class_ $ H.ClassName "ion-pound" ] []
+        , HH.text $ "\160" <> tag
+        ]
+      ]
+
+    _ -> HH.div_ []
+  
+  
+  renderTags :: forall p. RemoteData String (Array String) -> H.HTML p Query
+  renderTags = case _ of
+    NotAsked ->  HH.div_ [ HH.text "Tags not loaded" ]
+    Loading -> HH.div_ [ HH.text "Loading Tags" ]
+    Failure err ->  HH.div_ [ HH.text $ "Failed loading tags: " <> err ]
+    Success tags ->
+      HH.div
+      [ HP.class_ CC.tagList ]
+      $ tags <#> (\tag ->
```

Comment:
Same comment about moving this to a helper function to avoid the odd layout here.